acmwebsite/templates/wiki_view.xhtml

```diff
@@ -0,0 +1,32 @@
+<html py:strip=""
+      xmlns:py="http://genshi.edgewall.org/"
+      xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <py:extends href="master.xhtml" />
+
+  <head py:block="head" py:strip="True">
+    <title>${tg.config.get('site.name')} Wiki - ${parts['title']}</title>
+  </head>
+
+  <body py:block="body" py:strip="True">
+    <div class="row">
+      <div class="col-md-12">
+        <h1 class="page-header">${parts['title']}</h1>
+        <div py:replace="Markup(parts['body'])" class="page-body">Page content will go here</div>
+      </div>
+    </div>
+    <div class="row">
+      <nav class="navbar navbar-default">
+        <ul class="nav navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="${tg.url('/wiki/history/%s' % pagename)}">History</a>
```

Comment:
Old style string formatting

Reply:
How would you recommend formatting these URLs instead? I tried just doing '/wiki/history/$pagename' put that seemed to mangle the URL pretty bad.

Reply:
This is now fixed - wiki pages now follow the example from other controllers of using '[url]/{}'.format(var)